lib/core/src/Cardano/DB/Sqlite.hs

```diff
@@ -0,0 +1,234 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE OverloadedLabels #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+{-# HLINT ignore "Redundant flip" #-}
+
+-- |
+-- Copyright: © 2018-2019 IOHK
+-- License: Apache-2.0
+--
+-- An implementation of the DBLayer which uses Persistent and SQLite.
+
+module Cardano.DB.Sqlite
+    ( SqliteContext (..)
+    , chunkSize
+    , dbChunked
+    , destroyDBLayer
+    , handleConstraint
+    , queryLogFunc
+    , startSqliteBackend
+    , transformTrace
+    , unsafeRunQuery
```

Comment:
Not sure that we need to export everything here :thinking: ? Maybe it's best if some stuff remains hidden inside the module. I'd say: remove anything that isn't required by a dependent module :no_mouth: